[IMP] base_edifact: support format latin-1

thienvh332 opened the pull request.

alexis-via committed:
Rename base_phone_business_document_import to base_business_document_import_phone for consistency

Add connector-telephony in oca_dependencies.txt

alexis-via committed:
import phonenumbers in try/except

alexis-via committed:
Add and update README.rst

Commit:
8.0 Add support for partner bank matching on invoice update (#6)

Add support for partner bank matching on invoice update (before, it was only supported on invoice creation)

Commit:
8.0 Use try/except when importing external libs (#8)

[FIX] LINT

Use try/except when importing external libs
Remove self.ensure_one() that has nothing to do in an api.model method

alexis-via committed:
Prepare v10 branch

Rename __openerp__.py to __manifest__.py and set installable to False

alexis-via committed:
Port base_business_document_import* to v10

Also port all the modules that generate the XML documents: account_invoice_ubl, account_invoice_zugferd, purchase_order_ubl and sale_order_ubl

oca-travis committed:
Update base_business_document_import_phone.pot

alexis-via committed:
Don't update last_run when download fails

When using auto invoice download, the cron is executed by default by admin user, which is above record rules: only handle download config in the company of the user and improve matching by always selecting obj in the same company (or company = False)

alexis-via committed:
Add ability to run cron invoice download from admin for all companies in multi-company environnement

Fix my previous commit on sequence field on invoice import config (caused a crash when creating a new record)
Usability improvements on download config

alexis-via committed:
[MIG] base_business_document_import from 11 to 12

Migrate base_business_document_import_phone from v10 to v12
Convert readme to the new format

alexis-via committed:
Use limit=1 where relevant

Improve code for shipping addresse matching
Improve tests

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update base_business_document_import_phone.pot

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (1 of 1 strings)

Translation: edi-12.0/edi-12.0-base_business_document_import_phone
Translate-URL: https://translation.odoo-community.org/projects/edi-12-0/edi-12-0-base_business_document_import_phone/de/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (1 of 1 strings)

Translation: edi-12.0/edi-12.0-base_business_document_import_phone
Translate-URL: https://translation.odoo-community.org/projects/edi-12-0/edi-12-0-base_business_document_import_phone/hr/

lmarion-source committed:
[IMP] Handling DespatchAdvice files in order to modify stock pickings

lmarion-source committed:
[IMP] Handling DespatchAdvice files in order to modify stock pickings

lmarion-source committed:
[IMP] Multiple PO in one dspatchAdvice : PO is at the line level

lmarion-source committed:
[IMP] Multiple PO in one dspatchAdvice : PO is at the line level

lmarion-source committed:
[IMP] Take lots into account

lmarion-source committed:
[IMP] Take lots into account

lmarion-source committed:
[FIX] assign stock moves after confirmation

simahawk committed:
sale_order_import: fix XMLSyntaxError handling

OCA-git-bot committed:
sale_order_import 14.0.1.2.1

alexis-via committed:
[MIG] base_business_document_import_phone to v14 (from v12)

alexis-via committed:
base_business_document_import_phone: black, isort, etc.

simahawk committed:
sale_order_import: _parse_xml handle NotImplementedError

When no handler is defined for the given document
NotImplementedError could be raised.

Handle this case properly when parsing.

OCA-git-bot committed:
sale_order_import 14.0.1.2.2

Commit:
[UPD] Update base_business_document_import_phone.pot

OCA-git-bot committed:
[UPD] README.rst

TDu committed:
sale_order_import: add line update values hook

simahawk committed:
sale_order_import: fix missing company default

SO and SOL creation should be done w/ all defaults loaded.
In particular, the company is not propagated directly to the SOL
when we play onchanges.

Hence, at least in our tests, it can happen
that the company is not set yet on the line
and the computation of the price via 'product._get_tax_included_unit_price'
is broken when calling 'res.currency._convert' w/ an empty company.

Error example::

    [...]/base/models/res_currency.py", line 192, in _convert
    assert company, "convert amount from unknown company"
    AssertionError: convert amount from unknown company

ATM I'm adding only the company since it turns out that
if we load all defaults w/ `order.default_get` it breaks price
computation.

simahawk committed:
sale_order_import: reduce _prepare_order complexity

simahawk committed:
sale_order_import: refactor tests setup

simahawk committed:
sale_order_import: test onchange validation

simahawk committed:
sale_order_import: order_file_change improve readability

simahawk committed:
sale_order_import: get_xml_files_from_pdf does not exists

simahawk committed:
sale_order_import: test _parse_xml

OCA-git-bot committed:
sale_order_import 14.0.1.2.3

simahawk committed:
sale_order_import: update autorship

simahawk committed:
sale_order_import: use pdf_helper

simahawk committed:
sale_order_import: add tests TODOs

simahawk committed:
sale_order_import: test parse_pdf_order

simahawk committed:
sale_order_import: re-organize test files

simahawk committed:
sale_order_import: refactor parse_order, de-dup code

OCA-git-bot committed:
sale_order_import 14.0.1.3.0

Commit:
[UPD] Update sale_order_import.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_order_import 14.0.1.4.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-14.0/edi-14.0-sale_order_import
Translate-URL: https://translation.odoo-community.org/projects/edi-14-0/edi-14-0-sale_order_import/

TDu committed:
sale_order_import: fix invoicing partner

OCA-git-bot committed:
sale_order_import 14.0.1.4.1

alexis-via committed:
[MIG] base_business_document_import_phone to v16

simahawk committed:
sale_order_import: propagate all available fields

Handle additional fields dynamically if available.
This way, if you add a field to a record and it's value is injected by a parser
you won't have to override  to let it propagate.

OCA-git-bot committed:
sale_order_import 14.0.1.4.2

simahawk committed:
sale_order_import: accept default order data via ctx

You can now pass 'sale_order_import__default_vals' key in the ctx
to pass order values down the stack.

OCA-git-bot committed:
sale_order_import 14.0.1.5.0

rmorant committed:
[14.0][MIG] sale_order_import_edifact & base_edifact

rmorant committed:
[FIX] base_edifact: test chore

OCA-git-bot committed:
[UPD] README.rst

Commit:
[IMP] sale_order_import: pre-commit stuff

Commit:
[IMP] sale_order_import_edifact: pre-commit stuff

Commit:
[MIG] sale_order_import_edifact: Migration to 16.0

Commit:
[MIG] sale_order_import: Migration to 16.0

Commit:
[IMP] sale_order_import: Don't support importing csv anymore

Commit:
[IMP] sale_order_import: Refactor code

- Add new selection import_type field for choosing the type to import
- Refactor _parse_file with import_type field
- Drop some not used method

bosd committed:
base_business_document_import: add support for price with tax included

Commit:
[IMP] sale_order_import: Add tests for create/update order button

OCA-git-bot committed:
Merge PR #813 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update base_edifact.pot

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[IMP] base_edifact: support format latin-1

nilshamerlinck commented:
> Supports loading other data formats
please be a bit more specific, if the commit name is vague it might be because you need to split it in smaller commits

nilshamerlinck commented on base_edifact/models/edifact.py:
```diff
@@ -172,16 +180,21 @@ def map2odoo_currency(self, seg):
         """
         product = seg[2]
         pct = product[1]
+        if not product[0] and pia[1][0]:
```
```suggestion
        # fallback on SA if no EAN given
        if not product[0] and pia[1][0]:
```

OCA-git-bot committed:
Merge PR #816 into 16.0

Signed-off-by yvaucher

Commit:
[UPD] Update sale_order_import.pot

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[IMP] base_edifact: Supports more datetime formats

thienvh332 committed:
[IMP] base_edifact: fallback if address information is missing

thienvh332 committed:
[IMP] base_edifact: fallback on SA if no EAN given

thienvh332 committed:
[IMP] base_edifact: Add price with Price qualifier is AAB

thienvh332 committed:
[IMP] base_edifact: Add new test file and add test with format latin-1

thienvh332 committed:
[IMP] base_edifact: Add test product with fallback on SA if no EAN given

thienvh332 committed:
[IMP] base_edifact: Add test with no unit price

thienvh332 committed:
[IMP] base_edifact: Add test with datetime

nilshamerlinck commented on base_edifact/tests/test_base_edifact.py:
```diff
@@ -23,6 +23,11 @@ def test_pydifact_obj(self):
         obj = self.base_edifact_model.pydifact_obj(edifact_docu)
         # [1]: to get the list messages, [0]: to get the first list value of the segments
         self.assertEqual(obj[1]["segments"][0]["BGM"][1], "1AA1TEST")
+        # Test with format latin-1
```
better having atomic tests: add `test_pydifact_obj_latin1()` for testing latin-1

nilshamerlinck commented on base_edifact/tests/test_base_edifact.py:
```diff
@@ -47,13 +52,28 @@ def test_map2odoo_product(self):
         seg = ("1", "", ["8885583503464", "EN"])
         product = self.base_edifact_model.map2odoo_product(seg)
         self.assertEqual(product["barcode"], "8885583503464")
+        # Test product with fallback on SA if no EAN given
```
same idea here: if it's something new -> new test function

alexis-via committed:
[FIX] account_invoice_facturx: crash on HS code

This is a forward-port of a fix in v14

OCA-git-bot committed:
Merge PR #838 into 16.0

Signed-off-by alexis-via

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[IMP] base_edifact: Add method get description

thienvh332 committed:
[IMP] base_edifact: Add unit test for test_map2odoo_description

thienvh332 committed:
[FIX] sale_order_import_edifact: Allow files without vat

thienvh332 committed:
[FIX] sale_order_import_edifact: fix get quantity form QTY segment

thienvh332 committed:
[FIX] sale_order_import_edifact: fix problem product price is not provided

thienvh332 committed:
[IMP] sale_order_import_edifact: Fallback on PIA (SA) when LIN is empty

thienvh332 committed:
[IMP] sale_order_import_edifact: support get product desciption from IMD segment

OCA-git-bot committed:
Merge PR #707 into 16.0

Signed-off-by alexis-via

Commit:
[UPD] Update account_invoice_facturx_py3o.pot

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[IMP] sale_order_import_edifact: add new data demo

thienvh332 committed:
[IMP] sale_order_import_edifact: add testcase for no ean in LIN segment

thienvh332 committed:
[IMP] sale_order_import_edifact: add testcase for no PRI segment

Commit:
[IMP] sale_order_import: add confirm_order flag to wizard

When the flag is enabled, the order will be automatically confirmed.
Mostly used when the wizard is used as a tool to import orders.

OCA-git-bot committed:
Merge PR #839 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update sale_order_import.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #835 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

nilshamerlinck committed:
[IMP] sale_order_import_edifact: support EDI files with no extension

OCA-git-bot committed:
[IMP] update dotfiles

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-16.0/edi-16.0-sale_order_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-sale_order_import/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (62 of 62 strings)

Translation: edi-16.0/edi-16.0-sale_order_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-sale_order_import/es/

thaolt99 committed:
[IMP] sale_order_import: warn user when no price is defined in the file

OCA-git-bot committed:
Merge PR #743 into 16.0

Signed-off-by alexis-via

Commit:
[UPD] Update base_business_document_import_phone.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-16.0/edi-16.0-base_business_document_import_phone
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-base_business_document_import_phone/

OCA-git-bot committed:
Merge PR #807 into 16.0

Signed-off-by etobella

Commit:
[UPD] Update base_business_document_import.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-16.0/edi-16.0-base_business_document_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-base_business_document_import/

thienvh332 committed:
[IMP] base_edifact: add export edifact method

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (39 of 39 strings)

Translation: edi-16.0/edi-16.0-base_business_document_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-base_business_document_import/es/

thienvh332 committed:
[IMP] base_edifact: add unit test for export edifact

thienvh332 committed:
[IMP] account_invoice_edifact: add module support export edifact format

OCA-git-bot committed:
Merge PR #853 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #821 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update sale_order_import_edifact.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #844 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update sale_order_import.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-16.0/edi-16.0-sale_order_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-sale_order_import/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (63 of 63 strings)

Translation: edi-16.0/edi-16.0-sale_order_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-sale_order_import/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (24 of 24 strings)

Translation: edi-16.0/edi-16.0-sale_order_import_edifact
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-sale_order_import_edifact/es/

tuantrantg committed:
[ADD] base_wamas_ubl: Add new module to parse between WAMAS document and UBL document

thienvh332 committed:
[IMP] despatch_advice_import: pre-commit stuff

thienvh332 committed:
[MIG] despatch_advice_import: Migration to 16.0

thienvh332 committed:
[UDP] despatch_advice_import: update unit test

thienvh332 committed:
[IMP] despatch_advice_import_ubl: pre-commit stuff

thienvh332 committed:
[MIG] despatch_advice_import_ubl: Migration to 16.0

thienvh332 committed:
[IMP] despatch_advice_import_ubl: add new context

nilshamerlinck committed:
[FIX] base_business_document_import tests to run with or without stock module installed

thienvh332 committed:
[IMP] despatch_advice_import: update readme

thienvh332 committed:
[IMP] despatch_advice_import: add handling for context

thienvh332 committed:
[IMP] despatch_advice_import: add raise error

OCA-git-bot committed:
Merge PR #884 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #879 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update despatch_advice_import.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #878 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update despatch_advice_import_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

jbaudoux committed:
[FIX] pdf_helper: multi-attachments

Replace code by odoo.tools.pdf

jbaudoux committed:
[IMP] pdf_helper: add pdf_embed_xml

jbaudoux committed:
[IMP] base_ubl: use pdf.helper.pdf_embed_xml

Mark _ubl_add_xml_in_pdf_buffer as deprecated
Mark _embed_ubl_xml_in_pdf_content

tuantrantg committed:
[UPD] Update .pre-commit-config.yaml

OCA-git-bot committed:
Merge PR #875 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update base_wamas_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[IMP] base_wamas_ubl: Add record_data_to_wamas method

tuantrantg committed:
[FIX] base_wamas_ubl: Fix grammar of WEAP

tuantrantg committed:
[FIX] base_wamas_ubl: Handle special case for KSTAUS

Special requirement for the Customer Delivery Preference flow. For each customer, we need to produce 2 KSTAUS telegrams: one with KstAus_LagIdKom = kMEZ, one with KstAus_LagIdKom = kPAR.

OCA-git-bot committed:
Merge PR #901 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

tuantrantg committed:
[ADD] base_wamas_ubl: Add menu entries to help debug

OCA-git-bot committed:
Merge PR #903 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update base_wamas_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[UPD] base_wamas_ubl: update wamas grammar

OCA-git-bot committed:
Merge PR #902 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update base_wamas_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #907 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #906 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[UPD] base_wamas_ubl: update address export

tuantrantg committed:
[IMP] base_wamas_ubl: Improve grammar of KST, KSTAUS and LST.
Update the mapping of Kst_SatzKz (KST) and Lst_SatzKz (LST).
Update the type of KstAus_RfNr (KSTAUS) from 'int' to 'str' and related test

thienvh332 committed:
[IMP] account_invoice_edifact: update address export

tuantrantg committed:
[FIX] base_wamas_ubl: Fix bug wamas2ubl

 AUSKQ has wrong value in 'IvAusk_ExtRef' field

tuantrantg committed:
[UPD] base_wamas_ubl: Update grammars ARTEAN, WEAK, WEAP

thienvh332 committed:
[IMP] base_wamas_ubl: Customer, Supplier export fields correction

tuantrantg committed:
[UPD] base_wamas_ubl: Edit ART grammar and fix parsing boolean value

tuantrantg committed:
[UPD] base_wamas_ubl: Edit ARTE grammar and add test parsing boolean value

OCA-git-bot committed:
Merge PR #912 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

alexis-via committed:
[FIX] account_invoice_facturx: crash when name is empty on invoice line

Make name of invoice line required in the view (only on invoice line, not on regular move line)

OCA-git-bot committed:
Merge PR #913 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[FIX] base_wamas_ubl: address export correction

OCA-git-bot committed:
Merge PR #916 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #854 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update account_invoice_edifact.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[IMP] base_wamas_ubl: Improve key in ART

tuantrantg committed:
[UPD] base_wamas_ubl: Update grammars and small fix

OCA-git-bot committed:
Merge PR #848 into 16.0

Signed-off-by alexis-via

OCA-git-bot committed:
[BOT] post-merge updates

tuantrantg committed:
[UPD] base_wamas_ubl: Update ART, AUSK, AUSP grammars

OCA-git-bot committed:
Merge PR #898 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

nilshamerlinck committed:
[FIX] base_wamas_ubl: res.partner.title issues

nilshamerlinck committed:
[FIX] base_wamas_ubl: addresses issues

thienvh332 committed:
[FIX] base_wamas_ubl: Update product export uom/packaging

OCA-git-bot committed:
Merge PR #918 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #922 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

jbaudoux committed:
[FIX] base_wamas_ubl: Update product export uom/packaging

jbaudoux committed:
[FIX] base_wamas_ubl: partner names

Commit:
base_wamas_ubl: change 'RxAusk_RahmenTourId_HostTourKz' to return 'ODOO'

jbaudoux committed:
[FIX] base_wamas_ubl: extref & kretk/p

jbaudoux committed:
base_wamas_ubl: code beautification

Simplify code, fix CLI, fix encodings, make package & fix imports
Rebuild samples results

OCA-git-bot committed:
Merge PR #915 into 16.0

Signed-off-by alexis-via

Commit:
[UPD] Update account_invoice_facturx.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-16.0/edi-16.0-account_invoice_facturx
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-account_invoice_facturx/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (32 of 32 strings)

Translation: edi-16.0/edi-16.0-account_invoice_facturx
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-account_invoice_facturx/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (34 of 34 strings)

Translation: edi-16.0/edi-16.0-despatch_advice_import
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-despatch_advice_import/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (34 of 34 strings)

Translation: edi-16.0/edi-16.0-base_wamas_ubl
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-base_wamas_ubl/es/

jbaudoux committed:
[FIX] base_wamas_ubl: PickingResponse

Message can start with any header telegram

jbaudoux committed:
base_wamas_ubl: use wamas uom instead of unece

jbaudoux committed:
base_wamas_ubl: Fix package parsing

OCA-git-bot committed:
Merge PR #930 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update base_wamas_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-16.0/edi-16.0-base_wamas_ubl
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-base_wamas_ubl/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (33 of 33 strings)

Translation: edi-16.0/edi-16.0-base_wamas_ubl
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-base_wamas_ubl/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: edi-16.0/edi-16.0-despatch_advice_import_ubl
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-despatch_advice_import_ubl/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: edi-16.0/edi-16.0-account_invoice_edifact
Translate-URL: https://translation.odoo-community.org/projects/edi-16-0/edi-16-0-account_invoice_edifact/es/

nilshamerlinck committed:
[IMP] base_wamas_ubl: KRETP telegram

thienvh332 committed:
[UPD] sale_order_import_edifact: Update EDIFACT mapping

OCA-git-bot committed:
Merge PR #942 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #943 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[IMP] base_wamas_ubl: update ID for reception and return UBL template

OCA-git-bot committed:
Merge PR #946 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

thienvh332 committed:
[IMP] base_edifact: Map PIA segment with product default_code
